CMakeLists.txt

```diff
@@ -539,9 +548,6 @@ IF ( HAVE_MPI )
   MESSAGE(STATUS "MPI runner is ${MPIEXEC}")
 ENDIF(HAVE_MPI)
 
-MESSAGE(STATUS "CXX compiler flags: ${CMAKE_CXX_FLAGS}")
```

Comment:
Sorry, I was not asking to remove these ones but only those INITIAL ones. Please recover this.

Reply:
I put them back with better output. Target properties can still change these.